Modules/Devices/src/Devices.Domain/Entities/Identities/Identity.cs

```diff
@@ -77,5 +87,6 @@ private void EnsureNoActiveProcessExists()
 
 public enum DeletionProcessStatus
 {
-    Approved
+    Approved,
+    WaitingForApproval
```

Comment:
Can you assign them numbers please? Otherwise it's too easy to change the order of the enum members and hence accidentally change the status of the DeletionProcess